components/constants.js

```diff
@@ -0,0 +1,10 @@
+const IMAGES_GASSTATION = {
+    GALP: "https://www.telefonodirecto.es/wp-content/uploads/2017/07/Tel%C3%A9fono-de-Galp-Espa%C3%B1a.png",
+    CARREFOUR: "https://www.bitrefill.com/content/cn/b_rgb%3AFFFFFF%2Cc_pad%2Ch_600%2Cw_1200/v1578400400/carrefour-france.jpg",
+    REPSOL: "https://www.pta.es/wp-content/uploads/2020/04/logo-repsol.jpg",
+    AVIA: "https://avia.ch/fileadmin/images/AVIA_HighRes-310_Mitgliedfirmen.jpg",
+    CEPSA: "https://alastria.io/wp-content/uploads/2019/04/Logo-CEPSA-Compa%C3%B1%C3%ADa-Espa%C3%B1ola-de-Petr%C3%B3leos.png",
+    CAMPSA:"https://www.telefonoasistencia.net/wp-content/uploads/2019/06/telefono-campsa.jpg"
+    }
+    
+export default IMAGES_GASSTATION  
```

Comment:
add blank line at the end of file

Reply:
export as non default because in a constant file you can have more that one thing to export



export const Images_GASSTATION.....

Reply:
ok, i corrected some small error from add blank line, after into footer i did the two nav list like header list page how you told me. The truth i'm optimizing so much my code. Thanks you nico